recipes/jmp/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "jmp" %}
+{% set version = "0.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmind/jmp/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4d242fb14502b15a7c072e112bdcd7cb5d8b373d9733162eea23e0b9b7dbb6d0
+
+build:
+  number: 0
+  skip: true  # [not linux]
```

Comment:
We can make this noarch, no? I'm guessing that the issue is a missing dep on Windows?

Reply:
@ocefpaf I made it `noarch`. Windows is missing a dependency and for Mac OS X there is an issue with the `jaxlib.so` which is already reported upstream. So from my perspective this is ready to be merged now.